### PR TITLE
Post Cross Thing Trigger

### PR DESCRIPTION
**Pull request by yongpingchen**
Add `target` of trigger to `CommandTriggerRequest`

### CODE REVIEW

**Comment by yongpingchen** [comment]
@nfukuzaki can you have a internal review of API design?

**Comment by nfukuzaki** [comment]
LGTM

**Comment by yongpingchen** [comment]
@nfukuzaki can you confirm the implementation?

File: test/large/APIAuthorTriggerTest.ts [hunk 1]
```diff
@@ -4,13 +4,25 @@ import {expect} from 'chai';
...
+} from '../../src/ThingIFSDK'
+
 declare var require: any
 let thingIFSDK = require('../../../dist/thing-if-sdk.js');
 
 describe("Large Tests for APIAuthor Trigger APIs:", function () {
 
     let user: KiiUser;
-    let au: any;
+    let au: APIAuthor;
```
Comment (rizakii) [inline test/large/APIAuthorTriggerTest.ts]: `au` is not readable variable naming would be better some meaningful word ie: 'auth'

**Pull request merged**